docs/api/modal.md

```diff
@@ -189,15 +189,17 @@ ion-modal.stack-modal {
 }
 ```
 
-### Theming
+import ThemeExample from '@site/static/usage/modal/styling/theming/index.md';
 
-TODO: Playground Example
+<ThemeExample />
 
 ### Animations
 
-#### Custom animations
+The enter and leave animations can be customized by using our animation builder and assigning animations to `enterAnimation` and `leaveAnimation`.
 
-TODO: Playground Example
+import AnimationsExample from '@site/static/usage/modal/styling/animations/index.md';
+
+<AnimationsExample />
```

Comment:
There's an error when opening the Angular Stackblitz (`Property 'modal' does not exist on type 'AppComponent'`).